### PR TITLE
Fix intel binary links in bootstrap script

### PR DESCRIPTION
**Pull request by jhand2**
Intel software download links for older versions of PSW and DCAP are dead. Fix the URLs and update the sha2 hashes.

Signed-off-by: Jordan Hand <jorhand@microsoft.com>

### CODE REVIEW

**Comment by jhand2** [comment]
bors try

**Comment by jhand2** [comment]
bummer

**Comment by radhikaj** [comment]
bors delegate

File: scripts/install-windows-prereqs.ps1 [hunk 1]
```diff
@@ -16,16 +16,16 @@ Param(
     [string]$NodeHash = 'F68B75EEA46232ADB8FD38126C977DC244166D29E7C6CD2DF930B460C38590A9',
     [string]$Clang7URL = 'http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe',
     [string]$Clang7Hash = '672E4C420D6543A8A9F8EC5F1E5F283D88AC2155EF4C57232A399160A02BFF57',
-    [string]$IntelPSWURL = 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15929/Intel%20SGX%20PSW%20for%20Windows%20v2.5.100.2.exe',
-    [string]$IntelPSWHash = '744B52C2DEDA3EC2AAC495A4F671FE574C506C35523028584A74469911FB0707',
+    [string]$IntelPSWURL = 'http://registrationcenter-download.intel.com/akdlm/irc_nas/16115/Intel%20SGX%20PSW%20for%20Windows%20v2.5.101.3.exe',
```
Comment (radhikaj) [inline scripts/install-windows-prereqs.ps1]: The following docs would also need to be changed:
docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md
docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

**Comment by jhand2** [comment]
Looks like @chrisrod-MSFT is handling this here: https://github.com/openenclave/openenclave/pull/2316

So I'll plan on abandoning this in favor of that PR

**Comment by andschwa** [comment]
@radhikaj ugh sorry we need to update bors (#2318) and then we can use `bors delegate+` or even just `bors d+` but with this version it only works with `bors delegate=some-user`.

**Comment by CodeMonkeyLeet** [comment]
Closed as superseded by #2316

**Pull request closed without merging**